jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java

```diff
@@ -34,6 +38,48 @@
  */
 public class ReproducibleLayerBuilder {
 
+  // Uses the current directory to act as the file input to TarArchiveEntry (since all directories
+  // are treated the same in TarArchiveEntry).
+  private static final File DIRECTORY_FILE = Paths.get(".").toFile();
```

Comment:
Move into `UniqueTarArchiveEntries`?  Make as a java doc and include a link to TarArchiveEntry constructor as a `@seealso`?